src/server/viper.ts

```diff
@@ -18,7 +18,7 @@ let connection: rpc.MessageConnection | undefined;
 
 try {
     const server = spawn(
-        '/nix/store/p4qnsh2pfcw444z0p0jji6rf0hl5r9wk-zulu17.34.19-ca-jdk-17.0.3/bin/java',
+        'java',
```

Comment:
Unfortunately this doesn't work in my setup :-(

Reply:
``` shell
$ jq .'"viperSettings.javaSettings"'.javaBinary ~/Library/Application\ Support/VSCodium/User/settings.json 
"/nix/store/p4qnsh2pfcw444z0p0jji6rf0hl5r9wk-zulu17.34.19-ca-jdk-17.0.3/bin/java"
```

Reply:
Maybe we could do something similar to [this code snippet](https://code.visualstudio.com/api/language-extensions/language-server-extension-guide#using-configuration-settings-in-the-server)?

Reply:
We pass paths in via `args` now.